### PR TITLE
refactor(cce/cluster): rename functions and remove deprecated pkg

### PR DESCRIPTION
**Pull request by Lance52259**
**What this PR does / why we need it**:
The function names are not by design and they use deprecated packages: fmtp and logp.

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note
1. rename the resource functions.
2. remove the deprecated packages and update the error message.
3. rename the resource files.
```

## PR Checklist

* [ ] Tests added/passed.
* [ ] Documentation updated.
* [ ] Schema updated.

## Acceptance Steps Performed

```
PENDING
```

### CODE REVIEW

File: huaweicloud/services/cce/resource_huaweicloud_cce_cluster.go [hunk 1]
```diff
@@ -663,17 +669,17 @@ func resourceCCEClusterV3Read(_ context.Context, d *schema.ResourceData, meta in
...
 	return nil
 }
 
-func resourceCCEClusterV3Update(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+func resourceClusterUpdate(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
 	config := meta.(*config.Config)
 	cceClient, err := config.CceV3Client(config.GetRegion(d))
 	if err != nil {
-		return fmtp.DiagErrorf("Error creating HuaweiCloud CCE Client: %s", err)
+		return diag.Errorf("error creating CCE v3 client : %s", err)
```
Comment (Jason-Zhang9309) [inline huaweicloud/services/cce/resource_huaweicloud_cce_cluster.go]: extra space

File: huaweicloud/services/cce/resource_huaweicloud_cce_cluster.go [hunk 2]
```diff
@@ -724,27 +730,27 @@ func resourceCCEClusterV3Update(ctx context.Context, d *schema.ResourceData, met
...
+	return resourceClusterRead(ctx, d, meta)
 }
 
-func resourceCCEClusterV3Delete(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+func resourceClusterDelete(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
 	config := meta.(*config.Config)
 	cceClient, err := config.CceV3Client(config.GetRegion(d))
 	if err != nil {
-		return fmtp.DiagErrorf("Error creating HuaweiCloud CCE Client: %s", err)
+		return diag.Errorf("error creating CCE v3 client : %s", err)
```
Comment (Jason-Zhang9309) [inline huaweicloud/services/cce/resource_huaweicloud_cce_cluster.go]: extra space

File: huaweicloud/services/cce/resource_huaweicloud_cce_cluster.go [hunk 3]
```diff
@@ -421,15 +427,15 @@ func resourceClusterMastersV3(d *schema.ResourceData) ([]clusters.MasterSpec, er
...
 	config := meta.(*config.Config)
 	cceClient, err := config.CceV3Client(config.GetRegion(d))
 	if err != nil {
-		return fmtp.DiagErrorf("Unable to create HuaweiCloud CCE client : %s", err)
+		return diag.Errorf("error creating CCE v3 client : %s", err)
 	}
 	icAgentClient, err := config.AomV1Client(config.GetRegion(d))
 	if err != nil {
-		return fmtp.DiagErrorf("Unable to create HuaweiCloud AOM client : %s", err)
+		return diag.Errorf("error creating AOM v1 client : %s", err)
```
Comment (Jason-Zhang9309) [inline huaweicloud/services/cce/resource_huaweicloud_cce_cluster.go]: extra space

File: huaweicloud/services/cce/resource_huaweicloud_cce_cluster.go [hunk 4]
```diff
@@ -421,15 +427,15 @@ func resourceClusterMastersV3(d *schema.ResourceData) ([]clusters.MasterSpec, er
 	return nil, nil
 }
 
-func resourceCCEClusterV3Create(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+func resourceClusterCreate(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
 	config := meta.(*config.Config)
 	cceClient, err := config.CceV3Client(config.GetRegion(d))
 	if err != nil {
-		return fmtp.DiagErrorf("Unable to create HuaweiCloud CCE client : %s", err)
+		return diag.Errorf("error creating CCE v3 client : %s", err)
```
Comment (Jason-Zhang9309) [inline huaweicloud/services/cce/resource_huaweicloud_cce_cluster.go]: extra space

File: huaweicloud/services/cce/resource_huaweicloud_cce_cluster.go (hunk 1 above)
Comment (Lance52259) [reply huaweicloud/services/cce/resource_huaweicloud_cce_cluster.go]: done.

File: huaweicloud/services/cce/resource_huaweicloud_cce_cluster.go (hunk 4 above)
Comment (Lance52259) [reply huaweicloud/services/cce/resource_huaweicloud_cce_cluster.go]: done.

File: huaweicloud/services/cce/resource_huaweicloud_cce_cluster.go (hunk 3 above)
Comment (Lance52259) [reply huaweicloud/services/cce/resource_huaweicloud_cce_cluster.go]: done.

File: huaweicloud/services/cce/resource_huaweicloud_cce_cluster.go (hunk 2 above)
Comment (Lance52259) [reply huaweicloud/services/cce/resource_huaweicloud_cce_cluster.go]: done.

File: huaweicloud/services/acceptance/cce/resource_huaweicloud_cce_cluster_test.go [hunk 5]
```diff
@@ -225,28 +224,28 @@ func testAccCheckCCEClusterV3Destroy(s *terraform.State) error {
...
 		if rs.Primary.ID == "" {
-			return fmtp.Errorf("No ID is set")
+			return fmt.Errorf("No ID is set")
 		}
 
 		config := acceptance.TestAccProvider.Meta().(*config.Config)
 		cceClient, err := config.CceV3Client(acceptance.HW_REGION_NAME)
 		if err != nil {
-			return fmtp.Errorf("Error creating HuaweiCloud CCE client: %s", err)
+			return fmt.Errorf("Error creating HuaweiCloud CCE client: %s", err)
```
Comment (ShiChangkuo) [inline huaweicloud/services/acceptance/cce/resource_huaweicloud_cce_cluster_test.go]:
```suggestion
			return fmt.Errorf("error creating CCE client: %s", err)
```

File: huaweicloud/services/cce/resource_huaweicloud_cce_cluster.go [hunk 6]
```diff
@@ -3,26 +3,32 @@ package cce
...
+
 	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/common"
 	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
 	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils"
-	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils/fmtp"
-	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils/logp"
 )
 
+// Deprecated function name, used only for G42 and OCB references.
+func ResourceCCEClusterV3() *schema.Resource {
```
Comment (ShiChangkuo) [inline huaweicloud/services/cce/resource_huaweicloud_cce_cluster.go:29]: we can modify the code in G42

File: huaweicloud/services/acceptance/cce/resource_huaweicloud_cce_cluster_test.go (hunk 5 above)
Comment (Lance52259) [reply huaweicloud/services/acceptance/cce/resource_huaweicloud_cce_cluster_test.go:248]: the message content will be modified in another PR.

Comment (Lance52259) [reply huaweicloud/services/acceptance/cce/resource_huaweicloud_cce_cluster_test.go]: done.

File: huaweicloud/services/cce/resource_huaweicloud_cce_cluster.go (hunk 6 above)
Comment (Lance52259) [reply huaweicloud/services/cce/resource_huaweicloud_cce_cluster.go:29]: The description of the deprecated function has been updated.

**Pull request merged**